.goreleaser.yaml

```diff
@@ -167,12 +229,13 @@ release:
     - [grafana/pyroscope](https://hub.docker.com/r/grafana/pyroscope/tags)
 
     ```bash
-      docker pull docker.io/grafana/pyroscope:{{ trimprefix .Tag "v" }}
+      docker pull {{ .Env.IMAGE_PREFIX }}{{ .ProjectName }}:{{ .Env.IMAGE_TAG }}
```

Comment:
Will this work when missing the `docker.io` prefix?

Reply:
Ok that works: https://github.com/grafana/pyroscope/actions/runs/20616364461/job/59209803166